lotus-soup/testkit/role_miner.go

```diff
@@ -114,7 +121,13 @@ func PrepareMiner(t *TestEnvironment) (*LotusMiner, error) {
 		return nil, err
 	}
 
-	// prepare the repo
+	// create the node
+	// we need both a full node _and_ and storage miner node
+	n := &LotusNode{}
+
+	m := &LotusMiner{LotusNode: n, t: t}
```

Comment:
We are overloading the term `miner`. I suggest we rename `LotusMiner` to `MinerNode`, to be explicit that this is not just the subsystem that produces blocks, but the full libp2p process/node.